CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ erlang Cookbook CHANGELOG
 =========================
 This file is used to list changes made in each version of the erlang cookbook.
 
+v1.5.7 (2014-09-11)
+-------------------
+Add an attribute use_distro_version for using local repo.
+
```

Comment:
Please remove the version bump. Oftentimes many PRs will be combined and released at once.